internal/wrapnix/wrapper.go

```diff
@@ -89,13 +89,19 @@ func CreateWrappers(ctx context.Context, devbox devboxer) error {
 //     So, the bin-wrappers need to use a symlink to the latest devbox binary. This
 //     symlink is updated when devbox is updated.
 func CreateDevboxSymlink() (string, error) {
+	// Get the symlink path; create the symlink directory if it doesn't exist.
 	curDir := xdg.CacheSubpath(filepath.Join("devbox", "bin", "current"))
 	if err := fileutil.EnsureDirExists(curDir, 0755, false /*chmod*/); err != nil {
 		return "", err
 	}
 	currentDevboxSymlinkPath := filepath.Join(curDir, "devbox")
 
-	devboxBinaryPath, err := os.Executable()
+	// Get the path to the devbox binary.
+	execPath, err := os.Executable()
+	if err != nil {
+		return "", errors.WithStack(err)
+	}
+	devboxBinaryPath, err := filepath.EvalSymlinks(execPath)
```

Comment:
This will fail if existing symlink is already a loop. I think in that case we should just remove the symlink and not create a new one.

Reply:
I didn't remove the symlink: i'm a little concerned that if it was previously created and was working fine, then by removing the symlink we may break some bin-wrappers (that are from a different project) have the symlinkDir in their PATH